frontend/themes/light.css

```diff
@@ -191,18 +191,20 @@
 
         /* code highlighting */
         --cm-color-editor-text: #41323f;
-        --cm-color-comment:     #e96ba8;
-        --cm-color-atom:        #815ba4;
-        --cm-color-number:      #815ba4;
-        --cm-color-property:    #b67a48;
-        --cm-color-keyword:     #ef6155;
-        --cm-color-string:      #da5616;
-        --cm-color-var:         #5668a4;
+        --cm-color-comment:     #b62a66;
+        --cm-color-keyword:     #b13e30;
+        --cm-color-symbol:      #9b570d;
+        --cm-color-command:     #607020;
+        --cm-color-macro:       #327644;
+        --cm-color-string:      #3a727b;
+        --cm-color-variable:    #5466a5;
+        --cm-color-literal:     #8e5189;
+        --cm-filter-type:        brightness(150%) saturate(50%);;
+
+        /* The colors below are not used in CellInput.js */
```

Comment:
can you update this comment? if they are not used then why have them?

Reply:
They're not used in `CellInput.js`, but some are used in things like `highlightjs.css`.

I'll try to refactor the highlightjs stuff before removing them.